### PR TITLE
updating log level for services

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Updated the log level in service from log to debug

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -10,7 +10,7 @@ DB_CONNECTION_ALIVE=true
 
 PORT=2015
 
-LOG_LEVEL=debug
+LOG_LEVEL=error
```
Comment (evgeny-s) [inline .env.local:13]: Let's revert to `debug` for localhost

File: src/modules/friendly-bot/friendly-bot.service.ts [hunk 2]
```diff
@@ -99,7 +99,7 @@ export class FriendlyBotService implements FriendlyBotServiceInterface {
   }
 
   private async getBalance(address: string, network: string): Promise<BalanceDto> {
-    this.logger.log(`About to get balance for: ${address}`);
+    this.logger.debug(`About to get balance for: ${address}`);
```
Comment (evgeny-s) [inline src/modules/friendly-bot/friendly-bot.service.ts:102]: Let's revert it to info

File: src/modules/health/health.service.ts [hunk 3]
```diff
@@ -52,7 +52,7 @@ export class HealthService {
   }
 
   public async blockProduction(): Promise<boolean> {
-    this.logger.log(`About to fetch block production time`);
+    this.logger.debug(`About to fetch block production time`);
```
Comment (evgeny-s) [inline src/modules/health/health.service.ts:55]: Let's revert it to info

File: src/modules/peers/peer.service.ts [hunk 4]
```diff
@@ -40,7 +40,7 @@ export class PeerService {
    * @returns peer details
    */
   public async fetch(network: string): Promise<any> {
-    this.logger.log(`About to fetch node details`);
+    this.logger.debug(`About to fetch node details`);
```
Comment (evgeny-s) [inline src/modules/peers/peer.service.ts:43]: Let's revert it to info

File: src/modules/block-scanner/block-scanner.service.ts [hunk 5]
```diff
@@ -60,14 +60,14 @@ export class BlockScannerService implements BlockScannerServiceInterface {
   }
 
   public startScanning() {
-    this.logger.log(`About to start scanning network`);
+    this.logger.debug(`About to start scanning network`);
```
Comment (evgeny-s) [inline src/modules/block-scanner/block-scanner.service.ts:63]: Let's revert it to info

File: .env.local (hunk 1 above)
Comment (evgeny-s) [reply .env.local:13]: Also - change LOG_LEVEL for `dev` and `prod`: It should be `INFO`

**Review by evgeny-s — CHANGES_REQUESTED** [review]

File: .env.local (hunk 1 above)
Comment (karthikeyan-cere) [reply .env.local:13]: done.

File: src/modules/block-scanner/block-scanner.service.ts (hunk 5 above)
Comment (karthikeyan-cere) [reply src/modules/block-scanner/block-scanner.service.ts]: reverted.

File: src/modules/peers/peer.service.ts (hunk 4 above)
Comment (karthikeyan-cere) [reply src/modules/peers/peer.service.ts]: reverted

File: src/modules/health/health.service.ts (hunk 3 above)
Comment (karthikeyan-cere) [reply src/modules/health/health.service.ts]: reverted

File: src/modules/friendly-bot/friendly-bot.service.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply src/modules/friendly-bot/friendly-bot.service.ts]: reverted

Comment (evgeny-s) [reply src/modules/friendly-bot/friendly-bot.service.ts]: I can see `debug. It should be something like:
`this.logger.info` or `this.logger.log`

File: src/modules/health/health.service.ts (hunk 3 above)
Comment (evgeny-s) [reply src/modules/health/health.service.ts]: I can see `debug`. It should be something like:
`this.logger.info` or `this.logger.log`

File: src/modules/peers/peer.service.ts (hunk 4 above)
Comment (evgeny-s) [reply src/modules/peers/peer.service.ts]: I can see `debug`. It should be something like:
`this.logger.info` or `this.logger.log`

File: src/modules/block-scanner/block-scanner.service.ts (hunk 5 above)
Comment (evgeny-s) [reply src/modules/block-scanner/block-scanner.service.ts]: I can see `debug`. It should be something like:
`this.logger.info` or `this.logger.log`

**Review by evgeny-s — APPROVED** [review]

**Pull request merged**